packages/renderer/src/lib/objects/KubernetesColumnName.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
```

Comment:
the `objects` directory may feel a bit meaningless? maybe something `shared` or `common`?

Reply:
https://github.com/podman-desktop/podman-desktop/tree/main/packages/renderer/src/lib/kube ?

Reply:
I added it here b/c the KubernetesObjectList and KubernetesObjectUI that it will always be used with are in this folder ... but I'm not sure why they weren't in /kube either. I'll move the column, but instead of:
  `/kube/KubernetesColumnName.svelte`
how about:
  `/kube/column/Name.svelte`
to match the proposal that @axel7083 had on column naming?

Reply:
I liked it, so new commit moves the column to `kube/column/Name.svelte`.